assets/js/modules/sign-in-with-google/utils/provisioning.js

```diff
@@ -46,11 +46,12 @@ export function sanitizeProvisioningParams( params ) {
 
 	// Start by replacing all invalid characters with hyphens in appname.
 	let sanitizedAppname = appname.replace( /[^a-zA-Z0-9\s-]/g, '-' );
-
 	// Reduce segments with 2+ consecutive spaces or hyphens to a single character.
 	sanitizedAppname = sanitizedAppname.replace( /[\s-]{2,}/g, ( match ) =>
 		match.includes( ' ' ) ? ' ' : '-'
```

Comment:
This could be simply truncated to 1 character rather than using a condition, no?

Reply:
We don't need to sanitize the appname property, i've removed this part.